src/Graph.cc

```diff
@@ -1039,31 +1083,31 @@ void Graph::removeNodes(std::function<bool(const Node*)> p) {
   }
 
   // -- remove references to nodes that satisfy p
-  for (auto& n : m_nodes) {
+  for (auto &n : m_nodes) {
     if (!n->isForwarding()) {
       removeLinks(&*n, p);
     }
   }
 
-  
   // -- remove nodes that satisfy p
 
   // (**) At this point we should have either unreachable nodes
   // satisfying p or unreachable nodes forwarding to nodes satisfying
   // p.
-  
+
   m_nodes.erase(
-  	std::remove_if(m_nodes.begin(), m_nodes.end(),
-  		       [p](const std::unique_ptr<Node, DsaAllocatorDeleter> &n) {
-			 // resolve node so that we also remove a node
-			 // if it's forwarding to a node that satifies
-			 // p.
-  			 return (p(n->getNode()));}),
-  	m_nodes.end());
+      std::remove_if(m_nodes.begin(), m_nodes.end(),
+                     [p](const std::unique_ptr<Node, DsaAllocatorDeleter> &n) {
+                       // resolve node so that we also remove a node
+                       // if it's forwarding to a node that satifies
+                       // p.
+                       return (p(n->getNode()));
+                     }),
+      m_nodes.end());
```

Comment:
there is a very large number of syntactic changes. This makes it very hard to carefully review the code. 
I find myself skipping big chunks because they seem to differ only in indentation. Would be good to split the PR into two: one for any formatting related changes and one for the actual content.